helmfile/values/rclone-sync.yaml.gotmpl

```diff
@@ -0,0 +1,36 @@
+{{- if or (ne .Values.objectStorage.type "s3") (ne .Values.objectStorage.sync.type "s3") }}
+{{- fail "rclone-sync is (for now) only supported using s3 on both ends" }}
+{{- end }}
+
+config:
+{{- if hasKey .Values.objectStorage.sync "dryrun" }}
+  dryrun: .Values.objectStorage.sync.dryrun
+{{- end }}
+  source:
+    name: default
+    type: {{ .Values.objectStorage.type }}
+    s3:
+      accessKey: {{ .Values.objectStorage.s3.accessKey }}
+      secretKey: {{ .Values.objectStorage.s3.secretKey }}
+      region: {{ .Values.objectStorage.s3.region }}
+      regionEndpoint: {{ .Values.objectStorage.s3.regionEndpoint }}
+      forcePathStyle: {{ .Values.objectStorage.s3.forcePathStyle }}
```

Comment:
I guess this should to be wrapped in a `{{ if eq .Values.objectStorage.type "s3" }}`
since we have the thing in the beginning of the file it shouldn't be an issue now, but I think it would look better to include it already